android-app/app/src/main/java/de/dhbw/ka/se/fibo/ApplicationState.java

```diff
@@ -40,7 +40,9 @@ private ApplicationState(Context context) {
         this.context = context;
 
         cashflows = new TreeSet<>();
-        populateTestData();
+        if (BuildConfig.DEBUG) {
+            populateTestData();
+        }
```

Comment:
I think these lines are not needed anymore. Anyways it'll probably not hurt if we keep it there as it might speed up our development process